doc/src/build/move/sui-move-library.md

```diff
@@ -0,0 +1,105 @@
+```
+let parent_info = object::new(ctx);
+let child = Child { info: object::new(ctx) };
+let (parent_id, child_ref) = transfer::transfer_to_object_id(child, parent_info);
+let parent = Parent {
+    info: parent_info,
+    child: child_ref,
+};
+transfer::transfer(parent, tx_context::sender(ctx));
+```
+To transfer an object `child` from one parent object to a new parent object `new_parent`, we can use the following API:
+```
+transfer::transfer_child_to_object(child, child_ref, &mut new_parent);
+```
+Note that in this call, we must also have the `child_ref` to prove the original ownership. The call will return a new instance of `ChildRef` that the new parent can maintain.
+To transfer an object `child` from an object to an account address `recipient`, we can use the following API:
+```
+transfer::transfer_child_to_address(child, child_ref, recipient);
+```
+This call also requires to have the `child_ref` as proof of original ownership.
+After this transfer, the object will be owned by `recipient`.
+
+More examples of how objects can be transferred and owned can be found in
+[object_owner.move](https://github.com/MystenLabs/sui/blob/main/crates/sui-core/src/unit_tests/data/object_owner/sources/object_owner.move).
+
+### Freeze an object
+To make an object `obj` shared and immutable, one can call:
+```
+transfer::freeze_object(obj);
+```
+After this call, `obj` becomes immutable which means it can never be mutated or deleted. This process is also irreversible: once an object is frozen, it will stay frozen forever. An immutable object can be used as reference by anyone in their Move call.
+
+### Share an object (experimental)
+This feature is still in development. It only works in Move for demo purpose, and doesn't yet work in Sui.
+
+To make an object `obj` shared and mutable, one can call:
+```
+transfer::share_object(obj);
+```
+After this call, `obj` stays mutable, but becomes shared by everyone, i.e. anyone can send a transaction to mutate this object. However, such an object cannot be deleted, transferred or embedded in another object as a field.
+
+Shared mutable object can be powerful in that it will make programming a lot simpler in many cases. However shared object is also more expensive to use: it requires a full sequencer (a.k.a. a consensus engine) to order the transactions that touch the shared object, which means longer latency/lower throughput and higher gas cost. One can see the difference of the two programming schemes between not using shared object vs using shared object by looking at the two different implementations of TicTacToe: [No Shared Object](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/games/sources/tic_tac_toe.move) vs. [Shared Object](https://github.com/MystenLabs/sui/blob/main/sui_programmability/examples/games/sources/shared_tic_tac_toe.move).
+
+## Transaction context
+`TxContext` module provides a few important APIs that operate based on the current transaction context.
```

Comment:
important operations/methods of the API? I think the API is what I installed - Sui. This is a method of the API?